packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts

```diff
@@ -219,7 +219,7 @@ export class AuditLogManager {
               <mj-body>
                 <mj-section>
                   <mj-column>
-                    <mj-image width="150px" src="https://graphql-hive.com/logo.png"></mj-image>
+                    <mj-image width="200px" src="https://the-guild.dev/graphql/hive/github-org-image.png"></mj-image>
```

Comment:
You're right that this image has very different dimensions. I think even 300px width would be reasonable.